app/components/views/PrivacyPage/Privacy/Privacy.module.css

```diff
@@ -35,4 +35,20 @@
 }
 .contentTitleButtonsArea {
   margin-left: auto;
-}
\ No newline at end of file
+}
+.logs{
```

Comment:
Most of this css is already defined for the log panels in the Help tab. I guess we want the appearance for this new log panel to be the same, so should the existing css be reused rather than duplicating it here?

Reply:
this is a good example of a good opportunity to wrap shared styling in a component, with `.less` files you would import the styling in both places and this can make it very tough to maintain, but with css module we should move shared styling to a separated component with it;s own css module and then use the component in as many places as we like.

Reply:
I added a TODO to create a shared component for logs and using it instead. Will create an issue as well

Reply:
awesome, thanks @vctt94